### ISSUE TITLE
Write blog post about internship

### ISSUE THREAD

**Issue by mcharg** [ONRR/onrr.gov-site#1996]

**Comment by mcharg** [comment]
@kmhtran has started a draft and hopes to have something ready to review this week.

**Comment by Maroyafaied** [comment]
@kmhtran I'm moving this issue to this sprint, please let me know once the draft is ready for review. Thanks!

**Comment by lpgoldstein** [comment]
Meeting  with @kmhtran on Wednesday 8/3 to do an interview style blog post.

**Comment by lpgoldstein** [comment]
blog post drafted and with Kaitlin now for revision

**Comment by lpgoldstein** [comment]
closing because the blog post is done. will create another issue for a later sprint to publish the blog post
